### PR TITLE
Room state 62

### PR DESCRIPTION
**Pull request by NoahBartell**
Puts room state icons in the game to show whether an enemy has been defeated or not
image

### CODE REVIEW

File: dunwoody_disaster/views/MapScreen.py [hunk 1]
```diff
@@ -127,6 +127,14 @@ def pixmap(self) -> QPixmap:
                 icon = QPixmap(DD.ASSETS["lock"]).scaledToWidth(100)
                 result = DD.overlay(result, icon, room)
 
+        for room in rooms["beaten"]:
+            icon = QPixmap(DD.ASSETS["completed"]).scaledToWidth(30)
+            result = DD.overlay(result, icon, room)
+
+        for room in rooms["all"] - rooms["beaten"]:
+            icon = QPixmap(DD.ASSETS["un-completed"]).scaledToWidth(30)
+            result = DD.overlay(result, icon, room)
```
Comment (chanson02) [inline dunwoody_disaster/views/MapScreen.py:136]: We probably shouldn't show the icon if there is an unbeaten boss and at least one unbeaten non-boss.

**Review by chanson02 — APPROVED** [review]
I love this!!!!!!!!! :fire: :man_technologist: :eagle:

**Review by MLHumphries — APPROVED** [review]
I like

**Comment by EngineeringDude123** [comment]
Very Cool, but I've yet to change the map to the new mainmap so these might have to be readjusted. That was my bad for not doing it sooner.

**Pull request merged**